pkg/plugins/platform/local.go

```diff
@@ -837,7 +854,8 @@ func (l *LocalApp) Down(removeData bool) error {
 				}
 			}
 		}
-		dir := filepath.Dir(l.AppConfig.DataDir)
+		// Convenience identifier
+		dir := l.AppConfig.DataDir
```

Comment:
Just wondering why the filepath.Dir() was removed here? Is DataDir already a fully rooted path?

Reply:
Wow, took a look and it seems to me that filepath.Dir() should *never* have been used here right? That removes a level from the DataDir? Could end up removing the user's *home* directory?

Reply:
filepath.Dir() was actually what was removing the tail end of the dataDir filepath. Not quite sure why but it was. For example if GlobalDdevDir was `/Users/Nick/.ddev` and I appended the site name to the end to make `/Users/Nick/.ddev/site-name`, filepath.Dir would trim that back to `/Users/Nick/.ddev` thus deleting every site's data. Very slippery slope and I agree that this situation needs to have an *air tight seal*.